lib/active_shipping/carriers/usps.rb

```diff
@@ -286,7 +286,9 @@ def canned_address_verification_works?
       </CarrierPickupAvailabilityRequest>
       EOF
       xml = Nokogiri.XML(commit(:test, request, true)) { |config| config.strict }
-      xml.at('/CarrierPickupAvailabilityResponse/City').text == 'SAN FRANCISCO' && xml.at('/CarrierPickupAvailabilityResponse/Address2').text == '18 FAIR AVE'
+      city = xml.at('/CarrierPickupAvailabilityResponse/City')
+      address = xml.at('/CarrierPickupAvailabilityResponse/Address2')
+      city && city.text == 'SAN FRANCISCO' && address && address.text == '18 FAIR AVE'
```

Comment:
FYI: you could also use `city.try(:text) == '...' && address.try(:text) == '...'`. Some people don't like `try`, so feel free to leave it like this.